compass/list.py

```diff
@@ -21,6 +22,7 @@ def list_cases(test_expr=None, number=None, verbose=False):
 
     verbose : bool, optional
         Whether to print details of each test or just the subdirectories
+        When applied to suites, verbose will list the tests in the suite
```

Comment:
```suggestion
        Whether to print details of each test or just the subdirectories.
        When applied to suites, verbose will list the tests in the suite.
```
If you have multiple sentences, the API documentation may put them on the same line (the line breaks here aren't necessarily retained) so it's important to punctuate.